src/coreclr/inc/daccess.h

```diff
@@ -561,17 +561,19 @@
 #ifndef NATIVEAOT
 #include <stdint.h>
 
+#if !defined(HOST_WINDOWS) && !defined(NATIVEAOT)
```

Comment:
```suggestion
#ifndef HOST_WINDOWS
```
This is under `#ifndef NATIVEAOT`